content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+(CNI) sehingga Pod dapat berkomunikasi satu sama lain.  
+DNS klaster (CoreDNS) tidak akan menyala sebelum jaringan dipasangkan.**
+
+- Perlu diperhatikan bahwa jaringan Pod tidak boleh tumpang tindih dengan jaringan _host_
+  manapun: kamu akan menemui beberapa masalah jika terjadi tumpang tindih.  
+  (Jika kamu menemukan adanya bentrokan antara jaringan Pod
+  pilihan _plugin_ jaringan dengan jaringan _host_, kamu harus memikirkan blok
+  CIDR yang cocok untuk digunakan, kemudian menggunakannya pada saat `kubeadm init` dengan
+  `--pod-network-cidr` dan sebagai pengganti pada YAML _plugin_ jaringan).
+
+- Secara bawaan, `kubeadm` mengatur klastermu untuk menggunakan dan melaksanakan penggunaan
+  [RBAC](/docs/reference/access-authn-authz/rbac/) (role based access
+  control).  
+  Pastikan _plugin_ jaringan Pod mendukung RBAC, dan begitu juga seluruh manifes
+  yang kamu gunakan untuk menyebarkannya.
+
+- Jika kamu ingin menggunakan IPv6--baik jaringan _dual-stack_, ataupun jaringan _single-stack_ IPv6
+  --untuk klastermu, pastikan _plugin_ jaringan Pod
+  mendukung IPv6.  
+  Dukungan IPv6 telah ditambahkan pada CNI sejak [v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0).
+
+{{< /caution >}}
+
+{{< note >}}
+Saat ini Calico adalah satu-satunya _plugin_ CNI yang dapat menerima uji e2e (_end-to-end_) oleh proyek kubeadm.
+Jika kamu menemukan isu terkait _plugin_ CNI kamu harus membuat tiket pada pelacak isu masing-masing _plugin_
+alih-alih pada pelacak isu kubeadm maupun kubernetes.
+{{< /note >}}
+
+Beberapa proyek eksternal menyediakan jaringan Pod Kubernetes menggunakan CNI, beberapa di antaranya juga
+mendukung [Network Policy](/docs/concepts/services-networking/networkpolicies/).
+
+Lihat daftar
+[_add-on_ jejaring dan _network policy_](https://kubernetes.io/docs/concepts/cluster-administration/addons/#networking-and-network-policy) yang tersedia.
+
+Kamu dapat memasang _add-on_ jaringan Pod dengan perintah berikut pada simpul
+_control-plane_ atau simpul yang memiliki kredensial kubeconfig:
+
+```bash
+kubectl apply -f <add-on.yaml>
+```
+
+Kamu hanya dapat memasang satu jaringan Pod per klaster.
+Di bawah ini kamu dapat menemukan instruksi instalasi untuk beberapa _plugin_ jaringan Pod yang populer:
+
+{{< tabs name="tabs-pod-install" >}}
+
+{{% tab name="Calico" %}}
+[Calico](https://docs.projectcalico.org/latest/introduction/) merupakan penyedia jejaring dan _network  policy_. Calico mendukung sekumpulan opsi jejaring yang fleksibel sehingga kamu dapat memilih opsi yang paling efisien untuk situasimu, termasuk jaringan _non-overlay_ dan _overlay_, dengan atau tanpa BGP. Calico menggunakan mesin yang sama untuk melaksanakan _network policy_ pada _host_, _Pod_, dan (jika menggunakan Istio & Envoy) aplikasi yang berada pada lapisan _service mesh_. Calico bekerja pada beberapa arsitektur, meliputi `amd64`, `arm64`, dan `ppc64le`.
```

Comment:
```suggestion
[Calico](https://docs.projectcalico.org/latest/introduction/) merupakan penyedia jejaring dan _network  policy_. Calico mendukung sekumpulan opsi jejaring yang fleksibel sehingga kamu dapat memilih opsi yang paling efisien untuk situasimu, termasuk jaringan _non-overlay_ dan _overlay_, dengan atau tanpa BGP. Calico menggunakan mesin yang sama untuk melaksanakan _network policy_ pada hos, Pod, dan (jika menggunakan Istio & Envoy) aplikasi yang berada pada lapisan _service mesh_. Calico bekerja pada beberapa arsitektur, meliputi `amd64`, `arm64`, dan `ppc64le`.
```

Reply:
Resolved in 3f0a13e59b9e4d1a6766d9bd0ca89fa899f895a4